adserver/tasks.py

```diff
@@ -55,3 +56,49 @@ def daily_update_geos(day=None):
             GeoImpression.objects.filter(pk=impression.pk).update(
                 **{impression_type: values["country__count"]}
             )
+
+
+@app.task()
+def daily_update_placements(day=None):
+    """
+    Update the Placement index each day.
+
+    :arg day: An optional datetime object representing a day
+    """
+    start_date = get_ad_day()
+    if day:
+        start_date = day.replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
This looks fine but does mean we're sending a complex object (datetime+timezone) through celery which might not be ideal. I think we had some issues related to this on RTD.

Reply:
Hrm yea -- we should probably just be passing in a string then calling `dateutil.parse` on it.

Reply:
Should be working now, tested it locally 👍